src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -1,125 +1,169 @@
 package core.basesyntax;
 
+import java.time.DateTimeException;
 import java.time.LocalDate;
 import java.time.LocalDateTime;
 import java.time.LocalTime;
 import java.time.OffsetDateTime;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.Locale;
 import java.util.Optional;
 
 public class JavaDateTimeApi {
+    private static final String timezoneUkraine = "+02:00";
+    private static final String FORMAT_D_MMM_YYYY = "d MMM yyyy";
+    private static final String FORMAT_DD_MMMM_YYYY_HH_MM = "dd MMMM yyyy HH:mm";
```

Comment:
these fields are used only in according methods, let's declare them like local in according methods

Reply:
@antoniukivan constants are great solution in this case
let's return to the previous version @YuraBondarenko